### ISSUE TITLE
Problems when reproducing the model

### ISSUE THREAD

**Issue by a18548128738** [LiuAoyu1998/STIDGCN#10]
Why doesn't the gradient converge on pems08 datasets?
start training...
/home/liheng/gnn/STUP/STIDGCN/ranger21.py:131: UserWarning: This overload of addcmul_ is deprecated:
	addcmul_(Number value, Tensor tensor1, Tensor tensor2)
Consider using one of the following signatures instead:
	addcmul_(Tensor tensor1, Tensor tensor2, *, Number value) (Triggered internally at  /pytorch/torch/csrc/utils/python_arg_parser.cpp:1025.)
  exp_avg_sq.mul_(beta2).addcmul_(1 - beta2, grad, grad)
Iter: 000, Train Loss: 118.7926, Train RMSE: 145.8819, Train MAPE: 2.0570, Train WMAPE: 0.4951
Iter: 050, Train Loss: 223.1799, Train RMSE: 309.6898, Train MAPE: 2.4626, Train WMAPE: 0.9536
Iter: 100, Train Loss: 485.1459, Train RMSE: 688.3273, Train MAPE: 4.8120, Train WMAPE: 1.9848
Iter: 150, Train Loss: 2717.7112, Train RMSE: 3223.2273, Train MAPE: 39.2995, Train WMAPE: 11.8117
Epoch: 001, Training Time: 45.9079 secs
Epoch: 001, Inference Time: 5.4174 secs
Epoch: 001, Train Loss: 1010.2618, Train RMSE: 1237.8405, Train MAPE: 14.0598, Train WMAPE: 4.3740, 
Epoch: 001, Valid Loss: 6601.3602, Valid RMSE: 7668.9384, Valid MAPE: 99.7537, Valid WMAPE: 28.1916
###Update tasks appear###
Updating! Valid Loss: 6601.360203334263, epoch:  1
Iter: 000, Train Loss: 5834.1489, Train RMSE: 6794.1509, Train MAPE: 74.8438, Train WMAPE: 24.3131
Iter: 050, Train Loss: 47793.7305, Train RMSE: 61361.0781, Train MAPE: 702.6072, Train WMAPE: 204.2222
Iter: 100, Train Loss: 344498.3125, Train RMSE: 482951.6562, Train MAPE: 5207.2100, Train WMAPE: 1409.4125
Iter: 150, Train Loss: 1957044.1250, Train RMSE: 2565219.2500, Train MAPE: 33883.4141, Train WMAPE: 8505.6836
Epoch: 002, Training Time: 46.3692 secs
Epoch: 002, Inference Time: 5.4567 secs
Epoch: 002, Train Loss: 641819.2153, Train RMSE: 849191.9689, Train MAPE: 10615.1647, Train WMAPE: 2778.7050, 
Epoch: 002, Valid Loss: 3705713.2455, Valid RMSE: 4740337.4911, Valid MAPE: 62535.4806, Valid WMAPE: 15823.5952
No update

**Comment by kongkz** [comment]
我也遇到了同样的问题
